content/blog/embrace-kubernetes-part2/index.md

```diff
@@ -0,0 +1,69 @@
+---
+title: "It’s Time to Embrace Kubernetes! Really? "
+date: 2020-12-11
+meta_desc: "How to evaluate your choices when choosing Kubernetes as your application platform."
+meta_image: embrace-k8s.png
+authors:
+    - lee-briggs
+tags:
+    - kubernetes
+
+---
+
+In the first article in this series,  we gave you some questions to help you and others at your company decide if Kubernetes is right for you. In this post, we’ll give you an example of where Kubernetes can be a good fit.
```

Comment:
Link to the first article here?

Reply:
Was waiting for the first article to be published, otherwise the linkchecker becomes sad. and angry.